src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/SqlAction.cs

```diff
@@ -648,6 +649,125 @@ TraceId traceId
         return result;
     }
 
+    public async Task<CascadingUpdateResult?> CascadeUpdates(
```

Comment:
In general, I regard these cascade tickets as just the first step in getting cascade working. Longer-term, this SQL should be entirely pre-processable for performance. But getting it working here is the first step. I also think this shorter-term version is going to need ApiSchema.json support in places, but we'll discover that as we go.

Reply:
"pre-processable" sounds a lot like code generation...

Reply:
I'm thinking a JSON description of the cascade relationships that can be consumed by any backend.